backend/src/common/services/jwt/jwt.service.ts

```diff
@@ -18,11 +19,18 @@ class JwtService {
     }
 
     public async verifyToken(token: string): Promise<JWTPayload> {
-        const { payload } = await jwtVerify(
-            token,
-            new TextEncoder().encode(this.secretKey),
-        );
-        return payload;
+        try {
+            const { payload } = await jwtVerify(
+                token,
+                new TextEncoder().encode(this.secretKey),
+            );
+            return payload;
+        } catch (error) {
+            throw new HttpError({
+                status: HttpCode.UNAUTHORIZED,
+                message: (error as Error).message,
+            });
```

Comment:
You shouldn't throw an HttpError from this service. 
Instead, wrap the verify function in the plugin in a try-catch block and handle the error there.